src/scss/skin-modern/components/_controlbar.scss

```diff
@@ -7,6 +7,8 @@
   @include hidden-animated;
   @include layout-align-bottom;
 
+  visibility: unset !important;
```

Comment:
Is there any impact adding this? Maybe also for potential users who forked the UI.